### ISSUE TITLE
Incorporate study protocol from back-end changes

### ISSUE THREAD

**Issue by yago1994** [AppHatchery/Fabla-Front-end#470]
# Goal
For the psilocibin experiment, researchers want to have the ability to pause the participants study if the psilocibin dose session gets rescheduled. 

## Considerations
- The app should be able to pull the protocol again and refresh the protocol towards the future (starting 1 day in the future)
- The app should not overwrite the previous stored diaries or add 'missed diaries' from the past, it should just pull the upcoming diaries from the new protocol and make any adjustments to bonus thresholds, incentives, etc,. 
- We are expecting that the only items changing will be scheduling of when the diaries are delivered (datetime) & tentatively the question prompts (Strings)

## Potential implementations
1. On launch the app should check if the protocol version matches that available in Retool for that specific participant and re-download it if it doesn't, at this time we could fire a Track Event and display a Pendo guide as well to avoid needs for loading screens
2. We could use remote configuration settings to "tell" the app remotely to check for a new protocol the next time it opens

**Comment by yago1994** [comment]
Let's make sure to talk about this tomorrow during water cooler or Thursday @Wiza-Munthali (whenever you are free)

**Comment by Wiza-Munthali** [comment]
@yago1994 Has Charlotte already started working on these designs?

**Comment by yago1994** [comment]
@Wiza-Munthali I believe so! What we discussed was to put the content in a separate page that you can navigate to from Settings, not sure if you have any mock-ups to share @charlotteyao-emory

**Comment by Wiza-Munthali** [comment]
> [@Wiza-Munthali](https://github.com/Wiza-Munthali) I believe so! What we discussed was to put the content in a separate page that you can navigate to from Settings, not sure if you have any mock-ups to share [@charlotteyao-emory](https://github.com/charlotteyao-emory)

Hey @charlotteyao-emory any update on this?

**Comment by charlotteyao-emory** [comment]
> > [@Wiza-Munthali](https://github.com/Wiza-Munthali) I believe so! What we discussed was to put the content in a separate page that you can navigate to from Settings, not sure if you have any mock-ups to share [@charlotteyao-emory](https://github.com/charlotteyao-emory)
> 
> Hey [@charlotteyao-emory](https://github.com/charlotteyao-emory) any update on this?

@Wiza-Munthali Yes! here's the link to figma: 
https://www.figma.com/design/zDk3k6dkxpBl9Eg4C4cfCy/M3---Experiments?node-id=4250-9894&m=dev

**Comment by Wiza-Munthali** [comment]
> > > [@Wiza-Munthali](https://github.com/Wiza-Munthali) I believe so! What we discussed was to put the content in a separate page that you can navigate to from Settings, not sure if you have any mock-ups to share [@charlotteyao-emory](https://github.com/charlotteyao-emory)
> > 
> > 
> > Hey [@charlotteyao-emory](https://github.com/charlotteyao-emory) any update on this?
> 
> [@Wiza-Munthali](https://github.com/Wiza-Munthali) Yes! here's the link to figma: https://www.figma.com/design/zDk3k6dkxpBl9Eg4C4cfCy/M3---Experiments?node-id=4250-9894&m=dev

Thanks @charlotteyao-emory

**Comment by Wiza-Munthali** [comment]
@charlotteyao-emory minor question. Would you like the question title to wrap or ellipsis the text?

<img width="425" alt="Image" src="https://github.com/user-attachments/assets/740c722e-6adf-4355-bc0d-1a72676cbad4" />

<img width="425" alt="Image" src="https://github.com/user-attachments/assets/689136b4-537a-46d5-8750-d0fb676ba825" />

**Comment by yago1994** [comment]
@Wiza-Munthali from my perspective the wrapped is more usable (i.e. we might have similar worded questions where the end of the phrase distinguishes it, like asking entry and departure times)

**Comment by charlotteyao-emory** [comment]
@Wiza-Munthali  I agree with Yago that the wrapped style would make it more convenient for the users.
